### PR TITLE
Openvino GPU Unit/Python Tests fix failure 

### PR DESCRIPTION
**Pull request by sfatimar**
### Description
We fix iGPU Unit and Python tests with this PR
We add packaging pip pkg to build Many Linux DockerFile

### Motivation and Context
This change is required to make sure iGPU Unit Test/Python Tests with OV are fixed
 - If it fixes an open issue, please link to the issue here. -->

### CODE REVIEW

**Comment by sfatimar** [comment]
@jywu-msft If ORT 1.13 code freeze is still not done. We will appreciate this PR to go in too which addresses some iGPU unit test failures

File: tools/ci_build/github/linux/docker/scripts/requirements.txt [hunk 1]
```diff
@@ -9,3 +9,4 @@ argparse
 sympy==1.10.1
 flatbuffers
 protobuf==3.18.1
+packaging
```
Comment (jywu-msft) [inline tools/ci_build/github/linux/docker/scripts/requirements.txt:12]: is this needed for CI pipeline?

**Comment by jywu-msft** [comment]
> @jywu-msft If ORT 1.13 code freeze is still not done. We will appreciate this PR to go in too which addresses some iGPU unit test failures

i think we can get the iGPU test failure fixes in.  not sure about the other python/packaging related changes. can you provide more background context on the python/packaging changes and why they are required?

File: onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc [hunk 2]
```diff
@@ -87,7 +87,6 @@
         "^test_simple_rnn_batchwise_cpu",
         "^test_sub_uint8_cpu",
         "^test_mul_uint8_cpu",
-        "^test_add_uint8_cpu",
```
Comment (jywu-msft) [inline onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc:90]: removing this is causing failure for other pipelines it seems?

File: tools/ci_build/github/linux/docker/scripts/requirements.txt (hunk 1 above)
Comment (sfatimar) [reply tools/ci_build/github/linux/docker/scripts/requirements.txt:12]: yes

**Comment by sfatimar** [comment]
@jywu-msft Please review. This PR is critical to us to go into 1.13

**Review by jywu-msft — APPROVED** [review]

**Pull request merged**